### PR TITLE
Improve Windows Installer

### PR DESCRIPTION
**Pull request by mrpalide**
Did you run `make format && make check`? Yes

Fixes: -

 Changes:	
- improve skywire.bat
- add regenerating config for update skywire
- store logs for windows installer in `C:\Program Files\Skywire\local\logs`

How to test this PR:
- `make win-installer`, then install new version
- run by skywire shortcut on start menu or type in cmd/powershell
- see user message of app in Powershell like this
image
- check `C:\Program Files\Skywire\local\logs` for logs

Note: You can download `.msi` installer from this link too: https://drive.protonmail.com/urls/Q1S5Y6A8F8#3Sw4ftqrILMC

### CODE REVIEW

**Comment by mrpalide** [comment]
## ScreenCasts
Trying to make screencast for systray app on fresh OSs with multiple desktop
#### Ubuntu 22.04
OS : Ubuntu 22.04
Desktop: Gnome v 42.0
https://user-images.githubusercontent.com/79150699/167548571-f676e350-c53d-4da6-8b55-864dc857e2b7.mp4

#### TBD

**Comment by mrpalide** [comment]
## Systray on Linux
This PR is related to systray in Windows, but not bad if talk about systray on Linux.
You can download built binnary on current PR for Linux from [this](https://drive.protonmail.com/urls/57KYDH1CW8#OMd5QOCO8TtL) link.
This binaries built on EndeavourOS (Archbased).
**Gnome** is most integrated with systray and work fine without any problem.
**XFCE** is good, but sometime submenu never appear!
and finally, **KDE Plasma** is really strange and I can confirm that submenu (means menu of see and select sever for VPN client) not work.

### ScreenCast
I thougth should be good if I share screencast of systray functionality on some Linux distos with differente desktops.
#### Ubuntu
OS: Ubuntu 22.04 (Minimal Installation)
Desktop: Gnome 42.0

https://user-images.githubusercontent.com/79150699/167568719-9211a4c6-570d-43c5-8d81-73499bd43209.mp4

#### Nitrux (Debianbased)
OS: Nitrux 2.1.1
Desktop: KDE Plasma 5.24.4

https://user-images.githubusercontent.com/79150699/167568652-ef396dcb-4a39-42ca-b193-424b9e290699.mov

#### Manjaro (Archbased)
OS: Manjaro 21.2.6
Desktop: XFCE

https://user-images.githubusercontent.com/79150699/167568666-b2287651-70f4-4d41-9368-17e8e373f59a.mov

**Comment by ersonp** [comment]
image

getting this error on the `.msi` installer provided here https://drive.protonmail.com/urls/B2Y8X8TP8C#OZ48LfHXmCJv.

```
PS D:\Code\go\skywire> make win-installer
Building Systray Windows Binaries...
no Go files in D:\Code\go\skywire\cmd\apps
Build Complete!
Setting go-msi configuration...


    Directory: D:\Code\go\skywire\scripts\win_installer


Mode                 LastWriteTime         Length Name
----                 -------------         ------ ----
-a----         5/11/2022   3:08 PM              0 new.update
Configuration complete!
Building msi...
Rel: can't make D:\Code\go\skywire\scripts\win_installer\build\amd64\skywire-cli.exe relative to C:\Users\Erson\AppData\Local\Temp\go-msi351958395
mv : Cannot find path 'D:\Code\go\skywire\scripts\win_installer\skywire.msi' because it does not exist.
At D:\Code\go\skywire\scripts\win_installer\script.ps1:27 char:1
+ mv skywire.msi ../../skywire.msi
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (D:\Code\go\skyw...ler\skywire.msi:String) [Move-Item], ItemNotFoundException
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.MoveItemCommand
 
Build Complete!
```
This error on the build but I think it is my machine's issue; if I remember correctly I had an issue building previously too.

The logs are working.

File: pkg/visor/api.go [hunk 1]
```diff
@@ -540,12 +540,15 @@ func (v *Visor) GetAppError(appName string) (string, error) {
...
+	if v.procM != nil {
+		cSummary, err := v.procM.ConnectionsSummary(appName)
+		if err != nil {
+			return nil, err
+		}
 
-	return cSummary, nil
+		return cSummary, nil
+	}
+	return nil, nil
```
Comment (jdknives) [inline pkg/visor/api.go:551]: Should we return an error here if there is no `procM`?

**Review by jdknives — APPROVED** [review]

**Review by ersonp — APPROVED** [review]

File: pkg/visor/api.go (hunk 1 above)
Comment (mrpalide) [reply pkg/visor/api.go:551]: Actually no need for error. `procM` just sometimes start/initilaize with delay, and this is just checking for that, also if not start at all then visor not running with this error:
```
[2022-05-11T14:45:30+04:30] ERROR [visor:startup]: initializing module launcher returning error: lol
[2022-05-11T14:45:30+04:30] ERROR []: Failed to start visor.
```

**Pull request merged**